src/devtools/client/debugger/src/components/SourceOutline/getOutlineSymbols.tsx

```diff
@@ -50,7 +53,7 @@ function addHitCountsToFunctions(
 
   return functions.map(functionSymbol => {
     const hitCount = getClosestHitCount(hitCountsByLine, functionSymbol.location);
-    return { ...functionSymbol, hits: hitCount?.hits };
+    return Object.assign({}, functionSymbol, { hits: hitCount?.hits }); // { ...functionSymbol, hits: hitCount?.hits };
```

Comment:
can this be cleaned up?

Reply:
This was actually intentional.

In an earlier perf capture I noticed a bunch of `_objectSpread()` instances taking up a few milliseconds here, and I was wanting to see if a manual `Object.assign()` would improve this.

I don't know _why_ that would be getting transpiled, tbh. It shouldn't be given our Babel settings.

But at worst this shouldn't actually hurt.